src/workers/wallet.worker.js

```diff
@@ -0,0 +1,70 @@
+
+const ejsWallet = require('ethereumjs-wallet');
+const hdKey = require('ethereumjs-wallet/hdkey');
+const bip39 = require('bip39');
+
+const walletHdPath = "m/44'/60'/0'/0/0";
```

Comment:
to constants

Reply:
@masteralx this is not working for worker

Reply:
@masteralx modules and constants, which use "window" object,  now seaprated to windowModules.js